docs/topics/web-create-project.adoc

```diff
@@ -65,14 +65,26 @@ Registering the server path ensures that the {ProductShortName} server always us
 * *Ignore path*: Enter a path for files to exclude from analysis.
 * *Export CSV*: Exports the report data as a CSV file.
 * *Disable Tattletale*: Disables generation of a Tattletale report for each application.
-* *Class Not Found analysis*: Enables analysis of Java files that are not available on the class path. This option should not be used if some classes are unavailable for analysis.
+* *Class Not Found analysis*: Enables analysis of Java files that are not available on the class path.
++
+[NOTE]
+====
+This option should not be used if some classes are unavailable for analysis.
+====
+
```

Comment:
I broke out the second sentence as a note to be consistent with anayzeKnownLibraries; there's no change to the actual description of this option.